addons/mass_mailing/migrations/13.0.2.0/pre-migration.py

```diff
@@ -72,6 +72,29 @@
 ]
 
 
+def fill_mailing_subject(env):
```

Comment:
Is it normal that this function never gets called?
At least, it's not called by this file.

Reply:
Ouch, my bad. Thanks for checking. Added in 40745746543e553ac4b554d58d69c6cd60a4d86d